students/FyfyNguyen/session03/slicing_lab.py

```diff
@@ -1,8 +1,16 @@
 #!/usr/bin/env python3
 
 """
+<<<<<<< HEAD
```

Comment:
you got a bunch of merge conflicts here -- mostly jsut extra / missing newlines. not sure whay that is, but I think it's less liley to happen (and/or be easier to fix) if you do:

`git pull upstream master`

before pushing changes to your gitHub repo.

Reply:
But this has already been reviewed, so you can ignore it, too :-)